include/pangolin/utils/condition_variable.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <pangolin/utils/timer.h>
+
+#include <memory>
```

Comment:
Pangolin uses Boost if C++11 isn't available. We can't use unique_ptr, but shared_ptr is available via a compatibility header (https://github.com/stevenlovegrove/Pangolin/blob/master/include/pangolin/compat/memory.h)